testsuite/MDAnalysisTests/analysis/test_density.py

```diff
@@ -147,3 +151,29 @@ def test_density_from_Universe_update_selection(self):
             self.selections['dynamic'],
             self.references['dynamic']['meandensity'],
             update_selections=True)
+
+class TestGridImport(TestCase):
+
+    def setUp(self):
+        sys.modules.pop('MDAnalysis.analysis.density', None)
+
+    @block_import('gridData')
+    def test_absence_griddata(self):
+        # if gridData package is missing an ImportError should be raised
+        # at the module level of MDAnalysis.analysis.density
+        with assert_raises(ImportError):
+            import MDAnalysis.analysis.density
+
+    def test_presence_griddata(self):
+        # no ImportError exception is raised when gridData is properly
+        # imported by MDAnalysis.analysis.density
+
+        # mock gridData in case there are testing scenarios where
+        # it is not available
```

Comment:
It is a requirement in setup.py, maybe we should consider that it's always there